mir_eval/transcription.py

```diff
@@ -138,6 +138,173 @@ def validate(ref_intervals, ref_pitches, est_intervals, est_pitches):
                          "value")
 
 
+def validate_intervals(ref_intervals, est_intervals):
+    """Checks that the input annotations to a metric look like time intervals,
+    and throws helpful errors if not.
+
```

Comment:
I wasn't aware this function was already implemented :) That said, it doesn't check if `intervals.size > 0`, so an empty annotation with shape `(0, 2)` could still slip through it without issuing a warning. Perhaps the best option would be to add the warnings from my local implementation to `util.validate_intervals` and then use that one?